src/OpenColorIO/ops/lut1d/Lut1DOpCPU_SSE2.h

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright Contributors to the OpenColorIO Project.
+
+#ifndef INCLUDED_OCIO_LUT1DOP_CPU_SSE2_H
+#define INCLUDED_OCIO_LUT1DOP_CPU_SSE2_H
+
+#include <OpenColorIO/OpenColorIO.h>
+
+#include "CPUInfo.h"
+
+typedef void (Lut1DOpCPUApplyFunc)(const float *, const float *, const float *, int, const void *, void *, long);
```

Comment:
You mentioned that you were using a function point, just wondering if you thought of using std::function?
I think it is fair to use a function pointer here (for speed). The overhead from std::function might be unnecessary.

Reply:
`std::function` isn't something I've used really used before, so not sure what the benefit would be? I typically avoid fancy c++ features when focusing on performance, haha, I'll take a look.